src/99-appendices/04-entity-table.md

```diff
@@ -7,18 +7,26 @@ precede the reconstruction entity. REQUIRED and OPTIONAL entities for a given
-| Corresponding modality   | `mod-<label>`       |                                                                                                | OPTIONAL             |                             |                       |                        |                                                                            |               |                             |          |          |          |                            |                    |                  |                         |                          |                          |
-| Echo                     | `echo-<index>`      |                                                                                                |                      | OPTIONAL                    |                       |                        |                                                                            |               |                             |          |          |          |                            |                    |                  |                         |                          |                          |
-| Recording                | `recording-<label>` |                                                                                                |                      |                             | OPTIONAL              |                        |                                                                            |               |                             |          |          |          |                            |                    |                  |                         |                          |                          |
-| Processed (on device)    | `proc-<label>`      |                                                                                                |                      |                             | OPTIONAL              |                        |                                                                            |               |                             | OPTIONAL |          |          |                            |                    |                  |                         |                          |                          |
-| Space                    | `space-<label>`     |                                                                                                |                      |                             |                       |                        |                                                                            |               |                             |          |          |          |                            | OPTIONAL           | OPTIONAL         |                         | OPTIONAL                 |                          |
+A general introduction to entities is given in the section on
+[common principles](../02-common-principles.md#file-names-consist-of-key-value-pairs-entities)
```

Comment:
The anchor needs updating, and I think the text would work better matching the header.

```suggestion
[file name structure](../02-common-principles.md#file-name-structure)
```